api/index.php

```diff
@@ -43,8 +44,9 @@
 	} else if($data->url == "/user/login") {
 		$userController = new UsersController();
 		echo $userController->userLoginRequest($data);
+	} else if ($data->url == "/restaurant/review/save") {

```

Comment:
I think this endpoint on line 47 could be written better. Typically, 3 slashes can make it hard to read and understand. Perhaps we can use "restaurant/submit" instead of "/restauraunt/review/save". It is more transparent what the endpoint is doing and less words. I will also leave an article here for more information about RESTful endpoints: https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples/